From: Shawn Landden <shawnlandden@gmail.com>
Date: Tue, 3 Jul 2012 08:19:16 -0700
Subject: doc: index.html

fix up link in docs that otherwise needs index.html functionality
---
 doc/index.html |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/index.html b/doc/index.html
index b934dd9..17ff4bf 100644
--- a/doc/index.html
+++ b/doc/index.html
@@ -72,7 +72,7 @@
 <li><a href="spex.html">Fundamental Specifications</a></li>
 <li><a href="command.html">Specifications of Command Line Utilities</a></li>
 <li><a href="kyotoproducts.pdf">Presentation</a></li>
-<li><a href="api/">API Documents of the core library (C/C++)</a></li>
+<li><a href="api/index.html">API Documents of the core library (C/C++)</a></li>
 </ul>
 
 <ul>
-- 
